frontend/src/constants.ts

```diff
@@ -19,3 +19,5 @@ export const SUPPORT_ALBY_CONNECTION_NAME = `ZapPlanner - Alby Hub`;
 export const SUPPORT_ALBY_LIGHTNING_ADDRESS = "hub@getalby.com";
 
 export const SUBWALLET_APPSTORE_APP_ID = "uncle-jim";
+
+export const DEFAULT_MEMPOOL_URL = "https://mempool.space";
```

Comment:
Can this be removed? we always have a mempool URL set in the backend

Reply:
Done, but should we add a check in the backend to throw an error if required env variables are not set? I don't think we do this atm